Automattic/wp-calypso
# Jetpack Backups: Pre-filter backups activity log request

#### Changes proposed in this Pull Request

* This adds a backup action filter to the request so we have less results to parse

#### Testing instructions



* Visit a site with backups on both WordPress.com and Jetpack Cloud
* Working Backup screens should remain unchanged
* Sites with over 1000 requests in a day should display backups with this change

Example site that failed to display backups due to many entries in the activity log in a single day:

Before:
<img width="761" alt="Screen Shot 2021-12-15 at 4 12 01 PM" src="https://user-images.githubusercontent.com/1992658/146265648-af374e49-216e-43dd-b8e3-7f3f84204128.png">

After:
<img width="755" alt="Screen Shot 2021-12-15 at 4 10 49 PM" src="https://user-images.githubusercontent.com/1992658/146265497-9b6e824d-1e78-4f04-8b23-496e5799400d.png">
